src/content/docs/nrql/nrql-tutorials/nrql-tutorial-advanced-functions.mdx

```diff
@@ -240,9 +240,9 @@ You can also do simple matching. For instance, maybe you want to match a value t
 
 The regular expression engine uses RE2 syntax. If you need to delete characters, you may need to use the double backslashing escape sequences. (e.g. \\).
 
-Know that `RLIKE` has inherently more complexity and less performance than `LIKE`. Only use it when `LIKE` and other filtering functionality fails to fit your needs.       
+Remember that `RLIKE` has inherently more complexity and less performance than `LIKE`. Only use it when `LIKE` and other filtering functionality fails to fit your needs.       
     </Step>
-Regular expression support allows for near-infinite pattern matching possibilities. If you are already a Regex guru, you know the power this adds to NRQL. But if you're not, don't worry! There are tons of Regex resources available to reference online. Now that you've learned how to use Regex with NRQL, next on the list is using nested aggregation.
+Regular expression support allows for near-infinite pattern matching possibilities. If you are already a Regex guru, you know the power this adds to NRQL. But if you're not, don't worry! Regex has many resources available to reference online. Now that you've learned how to use Regex with NRQL, next on the list is using nested aggregation.
```

Comment:
regex should be lowercased, unless it's at the start of a sentence and etc. For example, see this Python language doc: https://docs.python.org/3/library/re.html

Reply:
Oooh good call, I didn't know this. Thanks very much, I've fixed the improperly capitalized one.